src/vaults/mainnet.json

```diff
@@ -280,6 +275,11 @@
       "slug": "kodiak-beraeth-stone",
       "stakeTokenAddress": "0x57161d6272F47cd48BA165646c802f001040C2E0"
     },
+    {
+      "beraRewardsVault": "0xBC8c3d0CecEf980E1f4A4775bd354614bE163556",
+      "slug": "kodiak-beramo-wbera",
+      "stakeTokenAddress": "0x7aEF016302C3a27417372CF3AA74A1708D1Ca291"
```

Comment:
Could you please add the token as well? See other Kodiak tokens for examples if needed. Some info can be found via https://berascan.com/address/0x7aEF016302C3a27417372CF3AA74A1708D1Ca291 if needed

Reply:
Ah, I missed that we need to add the token info for the stake token as well, sorry about that! Pushed the update; please let me know if that works now.